common-ui/src/main/res/values/design-system-colors.xml

```diff
@@ -0,0 +1,45 @@
+<!--
+  ~ Copyright (c) 2021 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+
+    <!-- Day / Light theme colors -->
+    <color name="primaryTextColor">@color/gray90</color>
+    <color name="secondaryTextColor">@color/gray70</color>
+    <color name="disabledColor">#59000000</color>
+
+    <color name="primaryButtonDefaultTextColor">@color/white</color>
+    <color name="primaryButtonHoverTextColor">@color/white</color>
+    <color name="primaryButtonPressedTextColor">@color/white</color>
+    <color name="primaryButtonDisabledTextColor">@color/disabledColor</color>
+
+    <color name="red50">#DE5833</color>
```

Comment:
Failing check because `red50` is also in `common-ui/src/main/res/values/colors.xml`